tests/test_ext_autosummary.py

```diff
@@ -74,7 +74,7 @@ def test_mangle_signature():
 
 
 def test_extract_summary(capsys):
-    settings = Mock(language_code='',
+    settings = Mock(language_code='en',
```

Comment:
Since v0.17, docutils starts to load language module on parsing reST document using StateMachine directly.  So far, our testcase uses invalid language code for testing autosummary.